doc/paradox/upgrade.md

```diff
@@ -29,6 +29,37 @@ Latest changes
 
 See @ref:[the generated tables of incompatible changes](compat-report.md)
 
+Upgrade from 3.3 to 3.4
+-----------------------
+
+The full list of changes and bug fixes in version 3.4 is [available on github](https://github.com/slick/slick/discussions/2344). 
+
+### Dependency upgrades
+
+Slick 3.4 updates the following upstream dependencies. If you use these libraries as part of your build, 
+you should update your dependency versions to match:
```

Comment:
Technically, that is unnecessary because eviction will kick in and override the lower explicit dependency version. But it would definitely be better practice so that the build file is accurate and not misleading.

That said, I think the bigger ramification is that if the user is relying on any behavior in those libraries that has changed, they need to be aware. That you linked to their changelog is super helpful for that!

Reply:
Yeah I wondered about this. It's possible that users might pin their own DB library versions (e.g. by excluding slick's transitive dependencies) so I thought I should call it out.